jira/client.py

```diff
@@ -2294,15 +2297,24 @@ def _create_oauth_session(self, oauth, timeout):
         self._session.verify = verify
         self._session.auth = oauth
 
-    def _create_kerberos_session(self, timeout):
+    def _create_kerberos_session(self, timeout, kerberos_options=None):
         verify = self._options['verify']
+        if kerberos_options is None:
+            kerberos_options = {}
 
         from requests_kerberos import HTTPKerberosAuth
-        from requests_kerberos import OPTIONAL
+        from requests_kerberos import OPTIONAL, DISABLED
+
+        mutual_authentication = OPTIONAL
+        if kerberos_options.get('mutual_authentication') == 'DISABLED':
+            mutual_authentication = DISABLED
+        else:
+            raise ValueError("Unknown value for mutual_authentication: %s" %
+                             kerberos_options['mutual_authentication'])
```

Comment:
Hey, we really want this behavior here?
If not set to DISABLED it will fail.

Reply:
Thanks for spotting this! You are very right.

Reply:
@pbabinca don't know if you want to work on a fix, but you could relate it to https://github.com/pycontribs/jira/issues/424. If so I can assign it to you.

Reply:
Yes, I will look into that one.